### ISSUE TITLE
Import of CSV files into a GraphCool Project

### ISSUE THREAD

**Issue by kdipaolo** [prismagraphql/graphcool-framework#548]
**Please post usage questions or broader discussions in the [Forum](https://www.graph.cool/forum/)**.

---

For **feature requests**, please fill in the next sections:

**What feature are you missing?**

**How could this feature look like in detail? Tradeoffs?**

---

For **bug reports**, please fill in the next sections:

**Current behavior**

I am trying to import a backup of data that I got from the graph cool (Prisma) team into a different project on the Graph Cool framework.  The backup is a zip folder with a bunch of CSV files. I don't see anywhere in the documentation on how to import CVS files via the command line.  

**Reproduction**

I've tried the following commands view the command line inside of my local graph cool project to import the data into my new project:

`graphcool import --data test.zip --target dev`
`graphcool import -d test.zip -t dev`

and this is the response i get:
`Unknown flag —data`

I also tried `graphcool import --source test.zip --target dev` and get the following:

`Invalid signature: 0x80014`

> If your problem can be reproduced with a certain service definition, please create a new GitHub repository with the reproduction instructions.

**Expected behavior?**

**Comment by divyenduz** [comment]
@kdipaolo : Can you validate that the CSV is in valid [NDF](https://www.graph.cool/docs/reference/data-import-and-export/normalized-data-format-teroo5uxih/) format? 

If no, we might have to convert it to NDF shape and import to work properly.

**Comment by kdipaolo** [comment]
@divyenduz I don't believe it is a valid NDF format, they are just plain CSV files.  How can I convert it to a valid NDF Shape?
